Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/utils/FillInStringGenerator.java

```diff
@@ -0,0 +1,19 @@
+package com.griddynamics.qa.vikta.uitesting.sample.utils;
+
+import java.util.UUID;
+
+public class FillInStringGenerator {
```

Comment:
maybe let's use more generic name like StringHelper. maybe we will put some other useful methods for strings  that  not only generate them